src/api/app/policies/appeal_policy.rb

```diff
@@ -0,0 +1,36 @@
+class AppealPolicy < ApplicationPolicy
+  def new?
+    create?
+  end
+
+  def show?
+    return false unless Flipper.enabled?(:content_moderation, user)
+    return true if record.appellant == user
+
+    user.is_admin? || user.is_moderator? || user.is_staff?
+  end
+
+  def create?
+    # TODO: Remove this block of comments
+    # I want to appeal a decision which is against me
+    # A decision is against me if:
+    # 1. I reported something, but the moderator cleared my report
+    # 2. Something I did/created was reported (comment, project, etc...), then the moderator favored the report
+    #
+    # If a report was created, we can safely assume the reporter had the rights to do so, otherwise the report wouldn't even exist in the first place
+    return false unless Flipper.enabled?(:content_moderation, user)
+
+    cleared_report_on_something_user_reported? || favored_report_on_something_user_can_update?
+  end
+
+  private
```

Comment:
```suggestion
  private

```

Reply:
Well, those changes are extracted to this https://github.com/openSUSE/open-build-service/pull/15342 and will go away once its merged and this PR rebased